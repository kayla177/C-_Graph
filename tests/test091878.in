ENTITY EPQJ9QBL name_EPQJ institution
ENTITY J1NXSFJK name_J1NX paper
ENTITY PM8IKIMS name_PM8I conference
ENTITY XHCJS2B1 name_XHCJ journal
ENTITY H0W9S9RD name_H0W9 conference
ENTITY 8IML64SK name_8IML conference
ENTITY R0TBTOZ5 name_R0TB author
ENTITY PU9R5IBF name_PU9R institution
ENTITY G9LSSOVF name_G9LS conference
ENTITY KBLJLUN3 name_KBLJ author
RELATIONSHIP PM8IKIMS published_in XHCJS2B1 5.0
RELATIONSHIP R0TBTOZ5 collaborated_with KBLJLUN3 4.3
RELATIONSHIP EPQJ9QBL presented_at KBLJLUN3 0.3
RELATIONSHIP KBLJLUN3 published_in PM8IKIMS 2.1
RELATIONSHIP H0W9S9RD cited_by R0TBTOZ5 4.4
RELATIONSHIP EPQJ9QBL authored_by XHCJS2B1 1.3
RELATIONSHIP XHCJS2B1 presented_at G9LSSOVF 4.9
RELATIONSHIP J1NXSFJK collaborated_with EPQJ9QBL 5.9
RELATIONSHIP KBLJLUN3 authored_by PU9R5IBF 8.3
RELATIONSHIP 8IML64SK affiliated_with R0TBTOZ5 7.9
RELATIONSHIP PU9R5IBF affiliated_with G9LSSOVF 9.6
RELATIONSHIP H0W9S9RD published_in J1NXSFJK 1.2
RELATIONSHIP 8IML64SK published_in XHCJS2B1 7.1
RELATIONSHIP H0W9S9RD affiliated_with PM8IKIMS 9.7
RELATIONSHIP J1NXSFJK collaborated_with KBLJLUN3 8.9
PATH EPQJ9QBL XHCJS2B1
DELETE PU9R5IBF
HIGHEST
PATH H0W9S9RD R0TBTOZ5
PRINT PM8IKIMS
HIGHEST
HIGHEST
HIGHEST
PRINT G9LSSOVF
PRINT G9LSSOVF
EXIT
