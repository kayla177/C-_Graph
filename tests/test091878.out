success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
EPQJ9QBL J1NXSFJK KBLJLUN3 PU9R5IBF G9LSSOVF XHCJS2B1 37.6
success
J1NXSFJK G9LSSOVF 33.1
H0W9S9RD R0TBTOZ5 4.4
XHCJS2B1 KBLJLUN3 H0W9S9RD
J1NXSFJK G9LSSOVF 33.1
J1NXSFJK G9LSSOVF 33.1
J1NXSFJK G9LSSOVF 33.1
XHCJS2B1
XHCJS2B1
